CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 * None.
 
 ### Bug fixes
-* Fixed a bug if `fullSynchronized` is not defined. This could lead to an exception when opening a synced Realm (#1989).
+* Fixed a bug if `fullSynchronization` is not defined in the configuration. This could lead to an exception when opening a synced Realm (#1989).
```

Comment:
"in the configuration" to what? :-)

Reply:
I have added a little more but we have *only one* configuration in Realm JS.